src/components/Navbar.tsx

```diff
@@ -188,6 +193,16 @@ export const Navbar = () => {
                         </div>
                         <ul className="flex flex-col items-start justify-start divide-y divide-charcoalBlack">
                             {currentUser && renderNavItems(true)}
+                            <a
+                                href="https://discord.com/invite/GxwvFEn9TB"
```

Comment:
The url needs to be updated.

Reply:
What should it be updated to? it still works for me, i just joined using the link.